[Instrumentation.GrpcCore] Update minimal Google.Protobuf to 3.15.0 (#1456)
open-telemetry/opentelemetry-dotnet-contrib: src/OpenTelemetry.Instrumentation.GrpcCore/CHANGELOG.md

  ([#483](https://github.com/open-telemetry/opentelemetry-dotnet-contrib/pull/483))
* Update OpenTelemetry.Api to 1.6.0.
  ([#1344](https://github.com/open-telemetry/opentelemetry-dotnet-contrib/pull/1344))

> * **Breaking Change** Drop Support to `Google.Protobuf` older than `3.15.0`.

Author: ```suggestion
*  Update minimal supported  version of `Google.Protobuf` to `3.15.0`.
```

> * **Breaking Change** Drop Support to `Google.Protobuf` older than `3.15.0`.

Author: Fixed
